sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/CommunityGalleryTests.cs

```diff
@@ -105,6 +105,8 @@ private void ValidateCommunityGalleryImage(CommunityGalleryImage communityGaller
         {
             string expectedId = "/CommunityGalleries/" + PublicGalleryName + "/Images/" + GalleryImageName;
             Assert.Equal(expectedId, communityGalleryImage.UniqueId);
+            Assert.NotNull(communityGalleryImage.Eula);
```

Comment:
Can you check for the exact expected value?

Reply:
Just in case something unexpectedly changes the value in an incorrect or breaking way.

Reply:
the exact value is a random string i put. It does not add much value here to check exact value because it might gets updated by mistake and this test will fail. But the only thing we want here is the string flow-through and NotNull assert do can 100% of that job right.